src/components/ReadingTime.vue

```diff
@@ -18,8 +18,8 @@ export default {
   },
   computed: {
     time() {
+      if (!this.text) return 1;
       const words = this.text.split(/\s/g).length;
-      console.log(words);
       return Math.ceil(words / this.WORDS_PER_MINUTE);
```

Comment:
can use ternary operator `return this.text ? Math.ceil(words / this.WORDS_PER_MINUTE) : 1;`